bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard
+
+openHAB dashboard is landing page for the user where all openHAB UIs can be found. Dashboard support also links to external services. Links can be added to the dashboard by editing the ```conf/services/dashboard.cfg``` configuration file.
```

Comment:
You can use single backticks when highlighting code inline (no need to have three backticks).